_posts/2016-01-20-release-add-ons-1.8.md

```diff
@@ -0,0 +1,129 @@
+---
+title:  "Release: Add-Ons 1.8"
+---
+
+We are proud to announce the availability of Add-On Self Administration 1.8 and Add-On Administration 1.8.
+This release mostly brings compatibility fixes to make the add-ons work with all versions of OSIAM >= 2.2 again.
```

Comment:
Shouldn't that be versions of OSIAM <= 2.3? That's at least what you said in all other documents.